x-pack/winlogbeat/module/security/config/winlogbeat-security.js

```diff
@@ -1316,6 +1475,15 @@ var security = (function () {
         })
         .Build();
 
+    var addComputerData = function(evt) {
+        var computer = evt.Get("winlog.computer_name");
+        if (!computer) {
+            return;
+    }
+        evt.Put("winlog.computer.name", computer.split(".")[0]);
```

Comment:
I would not make this assumption (and the next line) that these are the computer and domain by splitting `.` - because many organizations can/do have multi level domains (ie: addomain.mydomain.local)

Reply:
@neu5ron Does it make sense this?

evt.Put("winlog.computer.name", computer.split(".")[0]);
evt.Put("winlog.computer.domain", computer.split(".").slice(1).join('.');

If computer name is computer.addomain.mydomain.local

winlog.computer.name -> computer
winlog.computer.domain -> addomain.mydomain.local

Reply:
Many of these windows logs will already have the domain, and for the one's that don't - if the log does not have the domain, unless one can explicitly know with certainty what the domain is then I would not try to pull the domain from the computer name.. Otherwise there can be false/incorrect information..

Hope that makes sense, if not let me know :)

Reply:
@neu5ron you are correct. Thank you for your feedback.
If the computer is not joined or member of the domain I cannot fetch the domain name from the computer name.
In the case of the audit events like 1100, there is no other field in the event from where the domain name can be obtained; but to keep consistency and correctness I'll remove the addComputerData function

Reply:
@neu5ron @leehinman I have removed the addComputerData function.
Please let me know if I have to modify something else
Thank you!

Reply:
@janniten I think the PR needs to be rebased so there aren't merge conflicts and the CI tests will run.

Reply:
@leehinman . Done!

Reply:
@janniten sorry, I forgot to say "rebased off a more recent master" . It looks like this PR is still based on 9b48f2bbb2 from Wed Dec 18 11:43:56 2019 +0100.  Unfortunately 586eac17e8 on Fri Dec 20 13:54:27 2019 -0600 makes changes to the golden files in the same place this PR does, so they conflict.

Reply:
@leehinman Let me know I know is ok. Thank you!